workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardEmptyTable.tsx

```diff
@@ -45,7 +50,12 @@ export function DashboardEmptyTable({
 			</div>
 			<div>
 				{button && (
-					<ClayButton className="dashboard-empty-state-button">
+					<ClayButton
+						className="dashboard-empty-state-button mt-3"
+						onClick={() => {
+							navigate(`/order/${orderId}/create-license`);
+						}}
+					>
```

Comment:
This is wrong, we are using a generic component, used in others places adding a navigation capability to somewhere that probably:

Is not a order create license related

Reply:
@eliasantosf thank you for working on that, I already had this completed by my side since yesterday, sorry for not tell you earlier, here's the solution I made

https://github.com/brianchandotcom/liferay-portal/pull/144128/commits/4c10816888285ba201b2fa3bd3592b5c963b838e